Update pricing guide (#142)
2i2c-org/docs: about/pricing/strategy.md

- Be competitive with other "Data Science environment as a service" offerings (see below for how we consider ourselves relative to similar offerings).
- Be sustainable for the communities we serve, with mechanisms to accommodate institutions with fewer resources.

> ## Base fees for three service types
> 
> These are based on major use-cases in the communities we have served.
> 
> - **Educational communities** need basic hubs that are reliable, secure, and that serve a community of users. They tend to charge course fees and can recoup some costs this way. Assume 22 educational hubs per engineer.

Author: ```suggestion
## Pricing structure

Currently, we base our pricing on two major items:

### Flat monthly fees

We charge a flat monthly fee to cover [our personnel costs](costs/people.md).
We estimate the number of hubs an engineer can run, and use this to estimate our costs per hub after adding in project management and administration costs.

Most hubs take extra effort during the _set up_ phase, and relatively less effort to maintain over time (depending on how many change requests a community makes).
As such we suspect that this pricing structure does not cover our costs in the first month or two, but regains those costs in subsequent months.

In the future we may try to perform a more nuanced mapping of costs onto effort from our team, but for now we wish to keep things simple and predictable while we learn more.

### Pass-through cloud costs

In addition to our monthly hub fees, we pass cloud costs [directly to the communities we serve](costs/cloud.md), without taking any percentage markup.
We do this for two reasons:

1. In our eyes, we are running infrastructure _on behalf of each community_, and wish to act as if a member of that community were running the infrastructure themselves. We are simply being compensated for our time and expertise.
2. Adding a percentage markup on cloud costs may create perverse incentives for us to avoid optimizing down a community's cloud costs.

For these reasons, we are currently [passing through cloud costs directly to communities](costs/cloud.md).

## Base fees for three service types

These are based on major use-cases in the communities we have served.

- **Educational communities** need basic hubs that are reliable, secure, and that serve a community of users. They tend to charge course fees and can recoup some costs this way. Assume 22 educational hubs per engineer.
```

We do this for two reasons:

1. In our eyes, we are running infrastructure _on behalf of each community_, and wish to act as if a member of that community were running the infrastructure themselves. We are simply being compensated for our time and expertise.
2. Adding a percentage markup on cloud costs may create perverse incentives for us to avoid optimizing down a community's cloud costs.

For these reasons, we are currently [passing through cloud costs directly to communities](costs/cloud.md).

## Base fees for three service types

These are based on major use-cases in the communities we have served.

- **Educational communities** need basic hubs that are reliable, secure, and that serve a community of users. They tend to charge course fees and can recoup some costs this way. Assume 22 educational hubs per engineer.
- **Research communities** need a bit more power and complexity, and potentially access to more cloud infrastructure and data. They tend to have grant funding for fixed periods of time that is enough to cover an internal FTE. Assume 13 research hubs per engineer.

> - **Partnerships** happen with more advanced / well-resourced communities that want more bespoke infrastructure and new development, and we want a service option that allows for this flexibility. It takes significant extra effort to define and carry out these more complex relationships, and we should only engage in them if they cover a larger percentage of our costs. We'll use a lower-bound of at least 5 partnerships per engineer for these engagements.
> 

Author: ```suggestion
- **Partnerships** happen with more advanced / well-resourced communities that want more bespoke infrastructure and new development, and we want a service option that allows for this flexibility. It takes significant extra effort to define and carry out these more complex relationships, and we should only engage in them if they cover a larger percentage of our costs. We'll use a lower-bound of 20% FTE per engineer for these engagements (ie, at-most 5 partnership-level engagements per engineer), though we suspect most will require more FTE time than this.

```

- **Educational communities** need basic hubs that are reliable, secure, and that serve a community of users. They tend to charge course fees and can recoup some costs this way. Assume 22 educational hubs per engineer.
- **Research communities** need a bit more power and complexity, and potentially access to more cloud infrastructure and data. They tend to have grant funding for fixed periods of time that is enough to cover an internal FTE. Assume 13 research hubs per engineer.
- **Partnerships** happen with more advanced / well-resourced communities that want more bespoke infrastructure and new development, and we want a service option that allows for this flexibility. It takes significant extra effort to define and carry out these more complex relationships, and we should only engage in them if they cover a larger percentage of our costs. We'll use a lower-bound of 20% FTE per engineer for these engagements (ie, at-most 5 partnership-level engagements per engineer), though we suspect most will require more FTE time than this.

Below is a table which summarizes these major points

% Generated with https://www.tablesgenerator.com/markdown_tables

> | Type of service | Annual cost / engineer | % Eng. FTE | N Hubs / Eng. | Monthly fee | Annual fee total |
> |:---------------:|:----------------------:|:----------:|:-------------:|:-----------:|:----------------:|
> |    Education    |       $250,000.00      |    4-5%    |       22      |  $1,000.00  |    $12,000.00    |
> |     Research    |       $250,000.00      |    7-8%    |       14      |  $1,500.00  |    $18,000.00    |
> |   Partnership   |       $250,000.00      |    >=20%   |      <=5      |  $4,166.67  |   >=$50,000.00   |

Author: ```suggestion
| Type of service | Annual cost / engineer | % Eng. FTE | N Hubs / Eng. | Monthly fee | Annual fee total |
|:---------------:|:----------------------:|:----------:|:-------------:|:-----------:|:----------------:|
|    Education    |       $250,000.00      |    4-5%    |       22      |  $1,000.00  |    $12,000.00    |
|     Research    |       $250,000.00      |    7-8%    |       14      |  $1,500.00  |    $18,000.00    |
|   Partnership   |       $250,000.00      |    >=20%   |      <=5      | >=$4,166.67 |   >=$50,000.00   |
```
